jobs/config.json

```diff
@@ -11173,6 +11173,22 @@
       "sig-node"
     ]
   },
+  "pull-kops-bazel-build": {
+    "_comment": "NOTE: args for this are in prow/config.yaml after the `--` !",
+    "args": [],
```

Comment:
we do not need args?

Reply:
nope, they're inlined to prow/config.yaml. someday I will delete this file.
xref #4794

Reply:
we probably don't need a config.json entry in this case?

Reply:
there are presubmits that blow up without one